index.html

```diff
@@ -159,141 +156,401 @@ <h2>
-          <code>click</code>, but not <code>mousemove</code>. Engagement
-          gestures are any events included in the definition of being
-          <a>triggered by user activation</a> with the addition of
-          <code>keypress</code> and <code>keyup</code>. Note that operating
-          system level accessibility and internationalization features may
-          intercept gestures before the user agent processes them for
-          interaction with a page. E.g. multiple key press codes used to enter
-          an extended international character.
+          processed by the user agent to interact with the page. e.g. `click`,
+          but not `mousemove`. Engagement gestures are any events included in
+          the definition of being triggered by [=user activation=] with the
+          addition of `keypress` and `keyup`. Note that operating system level
+          accessibility and internationalization features may intercept
+          gestures before the user agent processes them for interaction with a
+          page. E.g. multiple key press codes used to enter an extended
+          international character.
         </dd>
       </dl>
     </section>
     <section>
       <h2>
-        <code>pointerlockchange</code> and <code>pointerlockerror</code> Events
+        Pointer Lock and Interfaces
       </h2>
-      <p>
-        Two events are used to communicate pointer lock state change or an
-        error in changing state. They are named <dfn>pointerlockchange</dfn>
-        and <dfn>pointerlockerror</dfn>. If pointer lock is entered or exited
-        for any reason a <a>pointerlockchange</a> event must be sent.
-      </p>
-      <p>
-        <a>User agents</a> must deliver these events by <a data-lt=
-        "queue a task">queuing a task</a> to <a>fire an event</a> of the
-        appropriate name with its <code>bubbles</code> attribute [[DOM]] set to
-        false to the pointer lock <a>target</a> element's [=Node/node
-        document=].
-      </p>
-      <p class="note">
-        Magnification software increases the size of content on the screen. It
-        uses the mouse to move the magnified point of focus around. When a
-        pointer lock is initiated, the magnification software needs to switch
-        to using the keyboard to move the magnified point of focus around
-        instead. When a pointerlockchange event is fired, web browsers
-        therefore need to make sure the event is communicated to assistive
-        technologies like screen magnifiers.
-      </p>
+      <section>
+        <h3>
+          The <dfn>pointer lock state</dfn> definition
+        </h3>
+        <p>
+          The [=pointer lock state=] is the state where a single DOM element,
+          which we will call the <dfn>pointer-lock target</dfn>, receives all
+          mouse events and the cursor is hidden.
+        </p>
+        <p>
+          Once in the [=pointer lock state=] the [=user agent=] has a
```

Comment:
can we split these definitions into three sentences maybe? There is too much information packed into this paragraph.

Reply:
How about,

```
    <section>
      <h2>
        Glossary
      </h2>
...
<dt>
          <dfn>Cursor position</dfn>
        </dt>
        <dd>
          Location of the system mouse cursor represented by the pair of
          {{MouseEvent/screenX}} and {{MouseEvent/screenY}}.
        </dd>
...
<section>
      <h2>
        Pointer Lock and Interfaces
      </h2>
      <section>
        <h3>
          The <dfn>pointer lock state</dfn> definition
        </h3>
        <p>
          The [=pointer lock state=] is the state when a single DOM element
          become <dfn>pointer-lock target</dfn>. The [=pointer-lock target=]
          receives all relevant user generated {{MouseEvent}} events: namely,
          all user-generated `mousemove`, `mousedown`, `mouseup`, `click`,
          `dblclick`, `auxclick`, and `wheel` [[ui-events]]. No other elements
          may receive these events while in [=pointer lock state=]. There will
          be no dispatching of events that require the concept of a mouse
          cursor: namely, `mouseenter`, `mouseleave`, `mouseover`, `mouseout`,
          `drag`, and `drop`.
        </p>
        <p>
          The boolean member {{PointerLockOptions/unadjustedMovement}} in
          {{PointerLockOptions}} can be used to change the mouse acceleration
          behavior in the [=pointer lock state=]. If
          {{PointerLockOptions/unadjustedMovement}} is `true`, the event
          coordinates must not be affected by the underlying platform behaviors
          such as mouse acceleration. In other words, UA must use the APIs
          provided by the underlying platform to guarantee getting the raw
          events. If {{PointerLockOptions/unadjustedMovement}} is `false`, UA
          must rely on the default behavior of the underlying platform
          regarding the mouse acceleration.
        </p>
```
What this does
 - move the <dfn>cursor position</dfn> into section of glossary.
 - move the MouseEvent description from original 2nd paragraph into 1st paragraph of definition.
 - refine original 3rd paragraph for description of PointerLockOptions/unadjustedMovement.

Also agree that we should drop the pointer-lock options concept and directly refer to PointerLockOptions.

Thanks,
Alvin

Reply:
yeah, that's better.